editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1668,6 +1668,10 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 				}
 			} break;
 			case MouseButton::RIGHT: {
+				if (preview_node->is_inside_tree() || spatial_editor->get_preview_material().is_valid()) {
+					break;
```

Comment:
I actually don't see the need for right-click to do anything in the viewport when drag-and-dropping.